### DISCUSSION TITLE
Credentials access issue

### DISCUSSION THREAD

**Discussion by gabrielserrao** [AntlerVC/firetable#1145] [Support / Q&A] [answered]
ROWY TRACING ID: ROWYERR-B8A8-08F9-F61E-E03AC153C65F

**Comment by harinij** [comment]
![Screenshot 2023-03-06 at 5 17 29 pm](https://user-images.githubusercontent.com/307298/223033303-d8677d1c-96c3-4c60-9c67-5c25f85f582d.png)

**Comment by BeeBombshell** [comment, marked as answer]
Hey @gabrielserrao, can you describe the steps which led you to this issue?

**Reply by BeeBombshell** [reply]
Assuming this issue has been resolved now, closing this one... @gabrielserrao if this is still an issue for you, feel free to create a new one.
